backend/backend/custom_settings.py

```diff
@@ -0,0 +1,9 @@
+'''
+This file contains granular settings specifically related to the Activists project.
```

Comment:
Quick note on this, @kurtisolo: we always do `activist` in lower case :) No stress though! I can fix 😊